### PR TITLE
Add Docker image & ci.jenkins.io update tasks

### PR DESCRIPTION
**Pull request by MarkEWaite**
Discovered during 2.263.3 release process.

### CODE REVIEW

**Review by markyjackson-taulia — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1-lts-release-checklist.md [hunk 1]
```diff
@@ -85,3 +85,6 @@ This role should rotate between LTS releases
 
 - [ ] Create pull request to update [configuration-as-code integration tests](https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/integrations/pom.xml) to the newly released version
 
+- [ ] Run trusted.ci.jenkins.io Docker image creation job ("Containers" -> "Core Release Containers" -> "master")
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:88]: could probably add a link for it?
```suggestion
- [ ] Run trusted.ci.jenkins.io [Docker image creation job](https://trusted.ci.jenkins.io:1443/job/Containers/job/Core%20Release%20Containers/job/master/)
```

File: .github/ISSUE_TEMPLATE/1-lts-release-checklist.md [hunk 2]
```diff
@@ -85,3 +85,6 @@ This role should rotate between LTS releases
...
+
+- [ ] [Update ci.jenkins.io](https://github.com/jenkins-infra/runbooks/tree/master/ci#upgrading-jenkins) to the new LTS release
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/1-lts-release-checklist.md:90]:
```suggestion
- [ ] [Update ci.jenkins.io](https://github.com/jenkins-infra/runbooks/tree/master/ci#upgrading-jenkins) to the new LTS release (note: repo is private, requires infra team membership)
```

**Review by timja — APPROVED** [review]
minor suggestions

**Review by oleg-nenashev — APPROVED** [review]
+1 to @timja's comments

**Review by timja — APPROVED** [review]

**Pull request merged**